.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+    "presets": ["next/babel"],
```

Comment:
Great work on creating the file called .babelrc at the root of our project. for the presets and plugins 🎨 ✨